tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/CaeAppTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Test.LabInfrastructure;
+using Microsoft.Identity.Test.Unit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Integration.NetFx.HeadlessTests
+{
+    [TestClass]
+    public class CaeAppTests
+    {
+        private string _confidentialClientSecret;
+        private readonly string _caeAppClientId = LabApiConstants.LabCaeConfidentialClientId;
```

Comment:
const? or don't define at all?

Reply:
When these are declared at the top, it's just easier to change them to my own config for testing.

Reply:
Afaik compiler compiles readonly primitives into const if it can anyway.